src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -422,6 +423,10 @@ public String getCookie() {
         }
 
         @Override public CauseOfBlockage getCauseOfBlockage() {
+            if (!FlowExecutionList.get().isResumptionComplete()) {
```

Comment:
Maybe use this test to gate https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/0a45db4d9db1da6ee5429ed742344f44f99ecd89/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java#L430-L435 but always do https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/0a45db4d9db1da6ee5429ed742344f44f99ecd89/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java#L436-L443 though #185 did introduce both blocks.

Reply:
See ef7ef1d.